### ISSUE TITLE
With recent chrome update tearing occurs where it had not in the past

### ISSUE THREAD

**Issue by doflo-dfa** [bvaughn/react-window#603]
This is a slightly confusing issue as my working virtualized lists have now become un-tethered to the scroll bar. 

React 17.02 
Mac Big Sur 11.5.2 
React-Window: 1.8.6

On 
Edge Version 95.0.1020.40 (Official build) (x86_64) & Chrome 93.0.4577.82 
the scrolling list will possibly render choppy but would never overscroll to nothing (meaning FPS suffers but it does not go "uncontrolled")

On 
Safari Version 14.1.2 (16611.3.10.1.6) & Chrome  95.0.4638.69 
the scroll bar is buttery smooth but can outpace the render which gets to a blank screen which then rendering catches up and appears, I have tested this returning a static ```<div>test</div>``` and in large jumps even that can get "over scrolled"

**Comment by doflo-dfa** [comment]
Chrome Version 96.0.4664.35 (Official Build) beta (x86_64)
seems to solve this problem... which is very odd.

**Comment by doflo-dfa** [comment]
Has anyone else seen this behavior ?

**Comment by fonty422** [comment]
I've definitely seen the tearing on Chrome. I noticed that the scrollbar tracked nicely on Firefox and there was no tearing. On Chrome I had both a mismatched scrollbar (i.e. the bar position and mouse position didn't match as per [this issue](https://github.com/bvaughn/react-window/issues/611) but also had tearing too. I did manage to fix the scrollbar issue by forcing the `estimatedItemSize` to a value that was equal to the minimum expected size, but the tearing persists. Is it a chrome thing, then, or a react-window issue?
